code/game/objects/structures/table_frames.dm

```diff
@@ -27,7 +27,10 @@
 
 ///Try to make a table with the item used to attack. FALSE if you can't make a table and should attack. TRUE does not necessarily mean a table was made.
 /obj/structure/table_frame/proc/try_make_table(obj/item/stack/stack, mob/user)
-	if(!stack?.table_type)
+	if(!isstack(stack))
```

Comment:
```suggestion
	if(!istype(stack))
```
you don't need to provide a type here, istype will check against the type the variable is typed to

Reply:
added